scripts/tasks/check-and-rewrite-package-json.js

```diff
@@ -38,16 +30,14 @@ const LWC_EXPOSED_MODULES = {
     ],
 };
 
-const directories = globSync('./packages/@lwc/*').filter(
-    (filename) => !IGNORED_PACKAGES.includes(filename.replace(/^packages\//, ''))
-);
-
 const expectedPkgJsons = [];
 
-for (const dir of directories) {
+for (const dir of globSync('./packages/@lwc/*')) {
     const filename = path.join('./', dir, 'package.json');
     const actual = fs.readFileSync(filename, 'utf-8');
     const pkg = JSON.parse(actual);
+    // Skip private packages
+    if (pkg.private) continue;
```

Comment:
```suggestion
    if (pkg.private) {
      continue;
    }
```

We don't enforce it, but we typically don't use the curly-bracket-less format. (We should add an ESLint rule for it.)